index.html

```diff
@@ -2,236 +2,338 @@
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="fourth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="fifth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="sixth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-    </section>
-
-    <section class="item-3">
-      <div class="about">
-        <h2 class="aboutMe">About me</h2>
-        <p class="decription">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <a class="seeProject">Get My Resume</a>
-        <img class="bottomline" src="assets/buttomline.png" alt="border line" />
-        <div class="language">
-          <div class="triangle">
-            <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle" />
-          </div>
-          <ul class="langElement">
-            <li class="lang">Language</li>
-            <li class="javascript">Javascript</li>
-            <li class="ruby">Ruby</li>
-            <li class="html">Html</li>
-            <li class="css">Css</li>
+    <main>
+      <section class="headline">
```

Comment:
- [ ] You did an amazing job adding the desktop version but kindly fix the design issue for screens from 600px to 992px because it is breaking. For example, you can set up a breakpoint but do not forget that you have to start again from that point because the style will lose or you can set up 2 breakpoints and add style for those screens. You can use this [link](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) if you need guidance for what I am saying. Keep coding 💪 
Here you have a screenshot of your project:

![image](https://user-images.githubusercontent.com/74506933/154802542-6e18820a-1bd4-4255-9ce0-6a4f2f1db9df.png)

![image](https://user-images.githubusercontent.com/74506933/154802550-b82488b5-54e6-4063-834a-8164c97bcc78.png)

![image](https://user-images.githubusercontent.com/74506933/154802554-11beee21-0100-46af-adb5-65a3b920ff2e.png)

![image](https://user-images.githubusercontent.com/74506933/154802562-51b01693-3d4b-4635-8a9a-6c159a41e546.png)

Reply:
but the requirement is for mobile and Destop only